protocols/luci-proto-pppossh/luasrc/model/network/proto_pppossh.lua

```diff
@@ -0,0 +1,48 @@
+-- Copyright 2018 Florian Eckert <fe@dev.tdt.de>
+-- Licensed to the public under the Apache License 2.0.
+
+local netmod = luci.model.network
+
+local proto = netmod:register_protocol("pppossh")
+
+function proto.get_i18n(self)
+	return luci.i18n.translate("PPPoSSH")
+end
+
+function proto.ifname(self)
+	return "pppossh-" .. self.sid
+end
+
+function proto.opkg_package(self)
+	return "pppossh"
+end
+
+function proto.is_installed(self)
+	return nixio.fs.access("/lib/netifd/proto/pppossh.sh")
+end
+
+function proto.is_floating(self)
+	return true
+end
+
+function proto.is_virtual(self)
+	return true
+end
+
+function proto.get_interfaces(self)
+	if self:is_floating() then
+		return nil
+	else
+		return netmod.protocol.get_interfaces(self)
+	end
+end
```

Comment:
Can this function simply return nil?  Or is that the default if we leave it out?

Reply:
Yes we have to return nil.
The luci-proto-qmi is doing the same so i will only return nil.
Thanks